src/Rings/AlgClosureFp.jl

```diff
@@ -20,6 +20,13 @@ import ..Oscar: algebraic_closure, base_field, base_ring, base_ring_type, charac
   elem_type, embedding, has_preimage_with_preimage, IntegerUnion, is_unit, map_entries,
   minpoly, parent_type, promote_rule, roots
 
+"""
+    AlgClosure{T} <: AbstractAlgebra.Field
+
+Type for the algebraic closure of a finite field.
+
+See [`algebraic_closure`](@ref).
+"""
 struct AlgClosure{T} <: AbstractAlgebra.Field
```

Comment:
This type probably should perhaps be renamed to `AlgClosureField`. Or perhaps `FqBarField` to match `QQBarField` ?

Any change should of course also be reflected in the name of `AlgClosureElem`.

Reply:
I agree. `AlgClosure` doesn't convey the meaning that it is about char p, and not the same as `QQBarField`.